jquery.facetview.js

```diff
@@ -1581,6 +1582,11 @@ search box - the end user will not know they are happening.
                     $(options.searchbox_class).bindWithDelay('keyup',dosearch,options.freetext_submit_delay);
                 }
 
+                // Properly set size for each facet
+                $.each(options.facets, function(facet){
+                  options.facets[facet]["size"] = options.facet_size;
```

Comment:
The problem with the change I made is that it alters the data in the application as a whole. This can be seen by the output of the plot when these lines are commented out. This is just one potential fix that I wanted you to look at, and I wanted to know if this change to the application is desirable.